docs/02.API-REFERENCE.md

```diff
@@ -3260,6 +3281,144 @@ jerry_set_object_native_handle (const jerry_value_t obj_val,
 
 - [jerry_create_object](#jerry_create_object)
 - [jerry_get_object_native_handle](#jerry_get_object_native_handle)
+- [jerry_set_object_native_pointer](#jerry_set_object_native_pointer)
+
+
+## jerry_get_object_native_pointer
+
+**Summary**
+
+Get native pointer and its type information. The pointer is previously associated with specified object
+Users can check the pointer's type before processing it.
+
+**Prototype**
+
+```c
+bool
+jerry_get_object_native_pointer (const jerry_value_t obj_val,
+                                 uintptr_t *out_native_p,
+                                 const jerry_object_native_info_t **out_info_p)
+```
+
+- `obj_val` - object value to get native pointer from.
+- `out_native_p` - native pointer (output parameter).
+- `out_info_p` - native pointer's type infomation (output parameter).
+- return value
+  - true, if there is native pointer associated with the object
+  - false, otherwise
+
+**Example**
+
+```c
+static void native_freecb (uintptr_t native_p)
+{
+  ... // free the native pointer
+}
+
+static const jerry_object_native_info_t type_info =
+{
+  .free_cb = native_freecb
+};
+
+{
+  jerry_value_t object;
+  uintptr_t native_set;
+
+  ... // receive or construct object and native_set value
+
+  jerry_set_object_native_pointer (object, native_set, &type_info);
+
+  ...
+
+  uintptr_t native_get;
+  const jerry_object_native_info_t *type_get_p;
+  bool is_there_associated_native = jerry_get_object_native_pointer (object, &native_get, &type_get_p);
+
+  if (is_there_associated_native)
+  {
+    if (out_info_p == &type_info)
+    {
+      ... // the type of object's native pointer is expected, and then process the native pointer.
+    }
+  }
+}
+```
+
+**See also**
+
+- [jerry_create_object](#jerry_create_object)
+- [jerry_set_object_native_pointer](#jerry_set_object_native_pointer)
+- [jerry_object_native_info_t](#jerry_object_native_info_t)
+
+
+## jerry_set_object_native_pointer
+
+**Summary**
+
+Set native pointer and an optional type information for the specified object
```

Comment:
Yes. But if your language differs too much from latin languages, it might be easy to forget these rules. I suspect capital letters do not exists in chinese. I am just curious about the full stops :)